src/domainobjectfactories/account_factory.py

```diff
@@ -187,11 +187,15 @@ def __create_closing_date(self, opening_date):
         Returns
         -------
         String
-            randomly generated date in YYYYMMDD format
+            randomly generated date in YYYYMMDD format, or "Empty"
         """
-        from_year = int(opening_date[:4])
-        from_month = int(opening_date[4:6])
-        from_day = int(opening_date[6:])
-        return self.create_random_date(from_year=from_year,
-                                       from_month=from_month,
-                                       from_day=from_day).strftime('%Y%m%d')
+
+        if random.getrandbits(1):  # fastest way to flip a coin
+            return "Empty"
+        else:
+            from_year = int(opening_date[:4])
+            from_month = int(opening_date[4:6])
+            from_day = int(opening_date[6:])
+            return self.create_random_date(
```

Comment:
I think everything you've added in fine, but I just took a look at the create_random_date method and I think the default closing date of 28/12/2017 is very arbitrary.  Is there a way you can have default closing date be today's date?